src/loaders/lottie/tvgLottieModel.h

```diff
@@ -130,6 +156,16 @@ struct LottieObject
         TVGERR("LOTTIE", "Unsupported slot type");
     }
 
+    virtual void copy(LottieSlot::Pair* pair)
```

Comment:
It's not a good idea because LottieObject (a pure base class) depends on the LottieSlot (an extension) feature. Even copying/reverting doesn't necessarily require a virtual and overriding approach, as the implementation can be common.

See: 
```
template<typename T>
assign(LottieSlot:Pair *pair, T& prop)
{
    delete(pair->prop);
    pair->prop = prop.copy();
}